generators/test/templates/template_test.go

```diff
@@ -1,58 +1,57 @@
 package test
 
 import (
-	"fmt"
-	"testing"
+  "fmt"
```

Comment:
Please run `go fmt` with this file. It will use TAB instead of SPACE, and automatically adjust the indentations.